### PR TITLE
Update wrangler to v1.1.0

### PR DESCRIPTION
**Pull request by rmweir**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
+	k8s.io/component-base => k8s.io/component-base v0.25.4
+	k8s.io/cri-api => k8s.io/cri-api v0.25.4
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.25.4
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.25.4
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.25.4
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.25.4
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.25.4
+	k8s.io/kubectl => k8s.io/kubectl v0.25.4
+	k8s.io/kubelet => k8s.io/kubelet v0.25.4
 	k8s.io/kubernetes => k8s.io/kubernetes v1.19.0
```
Comment (MbolotSuse) [inline go.mod]: Do we want to leave this one at 1.19 with all of the others at 0.25?

Comment (rmweir) [reply go.mod]: Fixed

**Review by maxsokolovsky — APPROVED** [review]
LGTM! Do we know for sure that Go 1.19 does not break anything with the adapter? If we haven't checked, then we might as well update Go to 1.20, since it's out. But it's up to you, as 1.19 is still supported.

File: go.mod [hunk 2]
```diff
@@ -61,21 +63,26 @@ require (
...
-	github.com/hashicorp/golang-lru v0.5.4 // indirect
 	github.com/imdario/mergo v0.3.12 // indirect
+	github.com/josharian/intern v1.0.0 // indirect
 	github.com/json-iterator/go v1.1.12 // indirect
+	github.com/mailru/easyjson v0.7.6 // indirect
 	github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369 // indirect
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
-	github.com/onsi/ginkgo v1.16.5 // indirect
+	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
```
Comment (MbolotSuse) [inline go.mod:85]: This library appears to have an odd license - https://github.com/munnerz/goautoneg/blob/master/LICENSE - are we ok with including this as a dependency?

**Review by MbolotSuse — COMMENTED** [review]
Tested basic functionality (go over licenses, confirm that we get a banner stating out of compliance and a support config configmap with the same information), and it looks good.

If we can confirm that the license of a new dependency is ok, I have no other objections.

File: go.mod (hunk 2 above)
Comment (MbolotSuse) [reply go.mod:85]: License appears to be 3-clause BSD, so we should be ok there. Resolving/approving on that basis.

**Review by MbolotSuse — APPROVED** [review]

**Pull request merged**